### DISCUSSION TITLE
my READ.ME damaged style

### DISCUSSION THREAD

**Discussion by RSS1102** [community/community#17036] [Other Feature Feedback, Questions, & Ideas] [answered]
Suddenly changed the style. The previous align = "right" for < img > has expired. I don't know what happened. Now I have to change the style again. Is it because of the problem of changing the resolution (everything was normal until last week).

``` 
<img width="400" align="right" src="https://pic.rmb.bdstatic.com/bjh/down/99b15364feac6f5cb587e8884b4361cc.gif" />
<img width="400" src="https://github-readme-stats.vercel.app/api/top-langs?username=RSS1102&show_icons=true&locale=en&layout=compact&line_height=20&title_color=f69673" alt="bongudth-stats" />
<img width="400"  src="https://github-readme-stats.vercel.app/api?username=RSS1102&show_icons=true&hide_border=false&line_height=20&title_color=f69673&icon_color=e2a7a2&show_owner=true" />

**Comment by imingx** [comment, marked as answer]
Here it is. #17256 

The reason is [`Option to prevent animated images from playing automatically`](https://github.blog/changelog/2022-05-19-option-to-prevent-animated-images-from-playing-automatically/).

**Reply by 0956397203** [reply]
ดี
